lib/pub_sub/active_record/events.rb

```diff
@@ -4,6 +4,9 @@ module PubSub
   module ActiveRecord
```

Comment:
any plans to add RuboCop to this repo?

Reply:
I find it more annoying than helpful in the other codebases we use it in, so I didn't set it up, but if you want, I can set it up. Should only take a few minutes.

Reply:
not a priority just wondering, we can do it later when it's more stable